### DISCUSSION TITLE
http://www.gstatic.com/generate_204 프로그램쓰다보니 어느순간 접속이안되고 위 링크가뜹니다

### DISCUSSION THREAD

**Discussion by oKaKao** [changdoc/naver-vaccine-macro#44] [Q&A]
차단당한건가요?

**Comment by changdoc** [comment]
처음 보는 증상이군요..

아래 블로그 글이 도움이 되실지 모르겠습니다.

https://iamfreeman.tistory.com/entry/%ED%81%AC%EB%A1%AC-%EB%B8%8C%EB%9D%BC%EC%9A%B0%EC%A0%80-redirect-wwwgstaticcomgenerate204-%EB%AC%B8%EC%A0%9C-%ED%95%B4%EA%B2%B0-%EB%B0%A9%EB%B2%95

**Comment by oKaKao** [comment]
해봤는데 안되네요ㅠㅠ

**Comment by Bi83** [comment]
아직 차단되었다는 말은 들은적이 없는데, 그냥 크롬 브라우저 문제가 아닐까요?
웨일브라우저로 시도해보세요.
